### PR TITLE
feature: Adds lsmy as a shortcut for lsgists from getenv defined user

### PR DESCRIPTION
**Pull request by vmesel**

### CODE REVIEW

**Comment by cuducos** [comment]
Thank you : )

Can we have `mygists` (as described in #13's title) instead of `lsmy`?

Reason: `lsmy` does not mean much in English and the pronunciation is bizarre (three consonants, no vowel to begin with). All the other shortcut commands, read out loud, are self-explanatory:

* `getmy init.vim` literally means _get my init.vim_ 
* `putmy init.vim` literraly means _put my init.vim_

So I think `mygists` would be just the user asking their gists, as in:
— hey, can you show _my gists_, please?

`ls` is indeed a well-known abbreviation for _list_, and could be used as in `lsmygists` — but it is too long and unpronounceable. `lsmy` doesn't say _what_ it's listing… _list my_… your what? sorry?

So my vote goes for `mygists`.

What do you think, @vmesel and @kmmelcher (who implemented the feature)?

File: README.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ How?
 
 1. Set an environment variable called ``GETGIST_USER`` with your GitHub
    username
-2. Use the shortcut ``getmy <filename>`` or ``putmy <filename>``
+2. Use the shortcut ``getmy <filename>``, ``putmy <filename>`` ``lsmy``
```
Comment (cuducos) [inline README.rst:133]:
```suggestion
2. Use the shortcut ``getmy <filename>``, ``putmy <filename>`` or ``lsmy``
```

File: getgist/__main__.py [hunk 2]
```diff
@@ -48,6 +48,11 @@
     Usage:  `lsgists <GitHub username>`.
 """
 
+LSMY_DESC = """
+    Call `lsgists` files assuming the user is set in an envvar called GETGIST_USER.
+    Usage:  `lsgists <GitHub username>`.
```
Comment (cuducos) [inline getgist/__main__.py:53]: Actually this line 53 doesn't make sense in this command helm text. Why suggesting `lsgists` here?

```suggestion
    See `lsgists --help` for more details.
```

**Review by cuducos — CHANGES_REQUESTED** [review]

**Comment by vmesel** [comment]
@cuducos makes lots of sense, actually! going to change its name on my codebase and update the PR.

File: getgist/__main__.py (hunk 2 above)
Comment (cuducos) [reply getgist/__main__.py:53]: Can you fix that too? Or did I get it wrong?

Comment (vmesel) [reply getgist/__main__.py:53]: Yes, I can fix it! Totally forgot about it! LOL

Comment (cuducos) [reply getgist/__main__.py]: Sorry, but 741256b  is not what I have suggested, and it actually breaks a pattern.

* `getmy --help` ends with _See `getgist --help` for more details._
* `putmy --help` ends with _See `putgist --help` for more details._

So I expected (as in my first comment) `mygists --help` to end with _See `lsgists --help` for more details._.

Does that make sense?

Comment (vmesel) [reply getgist/__main__.py]: Makes lots of sense! Going to update it!

**Review by cuducos — APPROVED** [review]

**Pull request merged**